### ISSUE TITLE
Incorrect route is used in getAllNodes() call

### ISSUE THREAD

**Issue by jpellizzari** [weaveworks/scope#2123]
Moved from https://github.com/weaveworks/service/issues/1053

The leading slash in the `topologyUrlsById` is causing the incorrect route to be requested.

Repro steps:
1. Within cloud, enter some search terms in scope.
2. `JSON.parse()` errors will appear in the console, as a result of a call to /api/topology/containers
3. Notice that the cloud index.html file is served instead of a JSON response.

The request to
https://cloud.weave.works/api/topology/containers
should be
https://cloud.weave.works/api/app/loud-breeze-77/api/topology/containers

**Comment by rade** [comment]
What functionality, if any, does this break? Search seems to work fine in Weave Cloud.

**Comment by jpellizzari** [comment]
Searching triggers a fetch of topologies which is throwing errors in the console. The search functionality itself seems to still work, since it does not rely on the result of the fetch. I'm not sure why searching needs to trigger that request, but its not working.

**Comment by rade** [comment]
> I'm not sure why searching needs to trigger that request,

This is worth investigating.

**Comment by jpellizzari** [comment]
@rade Found it!:

```javascript
    dispatch({ type: ActionTypes.FOCUS_SEARCH });
    // update nodes cache to allow search across all topologies,
    // wait a second until animation is over
    setTimeout(() => {
      getAllNodes(getState, dispatch);
    }, 1200);
```

We discussed a possible improvement in our cloud-prod sync today. I documented it here: https://github.com/weaveworks/service-conf/issues/500

**Comment by jpellizzari** [comment]
Closed by #2124
